### PR TITLE
Updated Chinese translation file (zh_CN.po)

### PR DESCRIPTION
**Pull request by wcxu21**
Updated Chinese translation file (zh_CN.po)

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -509,13 +538,21 @@ msgstr "粘贴之后(_A)"
 msgid "Paste _Before"
 msgstr "粘贴之前(_B)"
 
+#: data/menu.ui.h:26
+msgid "粘贴为奇数页(_O)"
+msgstr ""
```
Comment (dreua) [inline po/zh_CN.po:543]: I think you accidentally overwrote the msgid here.

Comment (dreua) [reply po/zh_CN.po:543]: Thats probably `msgid "Paste As _Odd Pages"`

File: po/zh_CN.po [hunk 2]
```diff
@@ -561,9 +598,13 @@ msgstr "放大(_I)"
 msgid "Zoom _Out"
 msgstr "缩小(_O)"
 
+#: data/menu.ui.h:41
+msgid "缩放适合(_F)"
+msgstr ""
```
Comment (dreua) [inline po/zh_CN.po:603]: I think you accidentally overwrote the msgid here.

Thats probably `msgid "Zoom _Fit"`

**Review by dreua — CHANGES_REQUESTED** [review]
Please review the two comments I made, otherwise it looks good, thank you!

File: po/zh_CN.po (hunk 2 above)
Comment (wcxu21) [reply po/zh_CN.po]: I'm so sorry, I've re-added the missing characters. Thanks for your thoughtful reminder.

**Comment by dreua** [comment]
Thank you!

**Pull request merged**